### PR TITLE
docs: add install by zip

### LINKED ISSUE #57: docs: Illustrate how to use the zip to install
https://docs.pgvecto.rs/getting-started/installation.html Here we could mention https://github.com/tensorchord/pgvecto.rs/releases/download/v0.2.1/vectors-pg14_x86_64-unknown-linux-gnu_0.2.1.zip and other zip packages.

### PR DESCRIPTION
**Pull request by cutecutecat**
Close #57

### CODE REVIEW

**Review by usamoi — COMMENTED** [review]
Have you tested it with CentOS?

**Comment by usamoi** [comment]
Glibc version in CentOS is too ancient...

**Comment by cutecutecat** [comment]
> Have you tested it with CentOS?

Changed to RHEL9, tested on AWS.

**Comment by usamoi** [comment]
Ubuntu 20.04 provides GLIBC 2.31, and RHEL 9 provides GLIBC 2.34 so it's acceptable. I download the zip and find the minimal supported GLIBC version should be 2.28 for now. I think we should document it.

Ref:
* https://distrowatch.com/table.php?distribution=ubuntu
* https://distrowatch.com/table.php?distribution=redhat

File: src/getting-started/installation.md [hunk 1]
```diff
@@ -54,6 +56,69 @@ DROP EXTENSION IF EXISTS vectors;
...
+```
+
+3. Download the zip package in the release page and extract it to a temporary directory.
+
+```sh
+wget https://github.com/tensorchord/pgvecto.rs/releases/download/v0.2.1/vectors-pg15_x86_64-unknown-linux-gnu_0.2.1.zip -O vectors.zip
+unzip vectors.zip -d vectors
+```
+
+4. Copy the extension files to the PostgreSQL directory.
```
Comment (usamoi) [inline src/getting-started/installation.md:96]: Users should pack the files to a package for their distros. Just copy the files is not a good idea.

Comment (cutecutecat) [reply src/getting-started/installation.md:107]: Many projects release a dynamic library in the form of a `.so` file, which users must copy to `/usr/lib`.

It's hard for us to instruct user to pack them, as different packer have different usages.

File: src/getting-started/installation.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ You can configure PostgreSQL by the reference of the parent image in https://hub
 
 ## From Debian package
 
+::: tip
+Installation from the Debian package requires a dependency on `GLIBC >= 2.29`, e.g:
```
Comment (usamoi) [inline src/getting-started/installation.md:40]:
```suggestion
Installation from the Debian package requires a dependency on `GLIBC >= 2.31`, e.g:
```

File: src/getting-started/installation.md [hunk 3]
```diff
@@ -58,12 +64,17 @@ CREATE EXTENSION vectors;
 
 ## From ZIP package
 
+::: tip
+Installation from the ZIP package requires a dependency on `GLIBC >= 2.29`, e.g:
```
Comment (usamoi) [inline src/getting-started/installation.md:68]:
```suggestion
Installation from the ZIP package requires a dependency on `GLIBC >= 2.31`, e.g:
```

**Pull request merged**